Documentation health metric (#307)
shadow-robot/sr-visualization: sr_health_check/README.md

## SR_HEALTH_CHECK PACKAGE

> Package that contains code of the Health Check GUI

Reviewer: ```suggestion
Package that contains the code of the Health Check GUI
```

> The purpose of the GUI is to allow the user running health checks of a Shadow Hand in order to get its health status. The information can be used to understand the current state of the hand any if there are any issues which might require support from the Shadow team.

Reviewer: ```suggestion
The purpose of the GUI is to allow the user to run health checks on a Shadow Hand in order to get its health status. The information can be used to understand the hand's current state and if any issues might require support from the Shadow team.
```

> The Run tab allows to select specific or all tests to be executed and stop the execution. The View tab presents the data collected for the health checks at different points of time. In case of a failed check, its font will be red coloured and expanding the tree with highlighted problems. 

Reviewer: ```suggestion
The `Run` tab allows the user to select which tests are to be executed and can also stop their execution. The `View` tab reports the data collected for the health checks at different points in time. In case of a failed check, its font will be red coloured and expanding the tree will allow the user to understand the cause of each problem. 
```

> Checks failing are not a unequivocal sign of the hand malfunctioning but in case of performance issues run the checks and contact the Shadow team for support.

Reviewer: ```suggestion
Checks failing are not an unequivocal sign of the hand malfunctioning, but in case of any performance issues, run the checks and contact the Shadow team for support.
```

> Every check class has a PASSED_THRESHOLDS class variable defininig the passing threshold for the checks, with the exact conditions being listed in the **has_passed** or **has_single_passed** method of the check.

Reviewer: ```suggestion
Every check class has a PASSED_THRESHOLDS class variable defining the passing threshold for the checks, with the exact conditions being listed in the `has_passed()` or `has_single_passed()` method of the check.
```
